TF/src/TFParser.py

```diff
@@ -79,15 +84,56 @@ def parse_graph(self, file_path, model_name, category,
             # Loop to populate to_nodes and from_nodes
             for operation in graph.get_operations():
 
+                # If graph contains StatefulPartitionedCall operation,
+                # only extracting the operations and returning empty graph
                 if operation.node_def.op == "StatefulPartitionedCall":
-                    print("Graphs with operation 'StatefulPartitionedCall' are not supported for parsing")
-                    return None
+                    print(
+                        "Graphs with operation 'StatefulPartitionedCall' are " 
+                        "not fully supported for parsing, graph or tensor " 
+                        "information not supported, only operators will be "
+                        "loaded into database."
+                        )
+
+                    NODES_DISCARDED = [
+                        "Const", "VarHandleOp", "StatefulPartitionedCall", 
```

Comment:
Any reason why this has been defined inside the loop? Not good from performance perspective?

Reply:
Will be defined only once as the function returns after entering condition. Since ops are discarded from this list only in case of 'StatefulPartitionedCall' operation being found, placed it inside the condition.